src/reqs.js

```diff
@@ -2,7 +2,9 @@ const express = require('express');
 const path = require('path');
 const exphbs = require('express-handlebars');
 const fs = require('fs');
+const controllers = require('./controllers');
+const { serverError, notFound } = require('./controllers/error');
 
-module.exports = {
-  express, path, exphbs, fs,
-};
+console.log(serverError, notFound);
```

Comment:
I removed this file entirely since noureldean didn't like it and you weren't comfortable using it as well.

Reply:
@FarahZaqout @NoureldeanSaed  deleted :+1: